htdocs/lg_config.php.example

```diff
@@ -24,6 +24,43 @@ $_CONFIG['logo'] = 'lg_logo.gif';
  */
 $_CONFIG['color'] = '#E48559';
 
+/*
+ * Enable reCAPTCHA v3
+ */
+$_CONFIG['recaptchaEnabled'] = false;
+
+/*
+ * reCAPTCHA v3 Frontend URL
+ */
+$_CONFIG['recaptchaFrontendURL'] = 'https://www.google.com/recaptcha/api.js?render=';
+
+/*
+ * reCAPTCHA v3 Backend Verify URL
+ */
+$_CONFIG['recaptchaBackendVerifyURL'] = 'https://www.google.com/recaptcha/api/siteverify';
+
+/*
+ * reCAPTCHA v3 Site Key
+ */
+$_CONFIG['recaptchaSiteKey'] = '';
+
+/*
+ * reCAPTCHA v3 Site Secret
+ */
+$_CONFIG['recaptchaSiteSecret'] = '';
+
+/*
+ * Should the LG show BGP peer information?
+ */
+$_CONFIG['showpeerinfo'] = 'TRUE';
+
+/*
+ * If 'showpeerinfo' above is TRUE, and you would like to show BGP peer information to visitors from a specific IPv4 address or subnet, add the IPv4 address or subnet (in CIDR format) to the array below
```

Comment:
Please clarify the comment, according to the code 'showpeerinfo' should be "FALSE" to filter peer info based on visitors IP.